src/osdag/design_type/compression_member/Column.py

```diff
@@ -865,19 +873,33 @@ def design_column(self):
         #     self.design_status = False
         #     self.design_status_list.append(self.design_status)
         self.epsilon = math.sqrt(250 / self.material_property.fy)
-        if len(self.input_section_list) > 0:
+        #if len(self.input_section_list) > 0:
```

Comment:
Again, comment.

Reply:
This was an unnecessary if condition for checking if user has added atleast one section as input and not left it as empty, but the GUI already gives us an error if the selected input section is left empty, so no need to add it to the logger. But again for debugging, I felt it was better to leave it out.